### ISSUE TITLE
[Feature Request] have it move to other monitor if you do hyprsome move onto same workspace

### ISSUE THREAD

**Issue by Kara-Zor-El** [sopa0/hyprsome#13]
Hi,

Really loving `hyprsome` but i would love to be able to move a window to the other monitor by doing `hyprsome move 1` whilst currently on the other monitors workspace 1

**Comment by Ultrahalf** [comment]
same can be done with shell script. But yeah that being included in hyprsome would be nice
switchwindowbetweenmonitor
```sh
#!/bin/bash

monitors=$(hyprctl monitors -j | jq '.[]')
if [ $(echo $monitors | jq '.id' | wc -l) -gt 1 ]; then
	passivemonitor=$(echo "$monitors" | jq 'select(.focused==false) | .id')
	hyprctl dispatch movewindow mon:$passivemonitor
	exit 0
fi

exit 0
```
```sh
bind = $mainMod SHIFT, comma,exec, switchwindowbetweenmonitor
```

**Comment by Ultrahalf** [comment]
works for 2 monitor. I would assume it will not work for 2+ monitor not sure

**Comment by sopa0** [comment]
Yeah I could implement that pretty easily but idk how it would work with 3 monitors

**Comment by Kara-Zor-El** [comment]
perhaps it'd cycle around. like move monitor 1 to 2 then 2 to 3 and etc... even having maybe a keybinding to swap monitor for it might be a good idea. like being able to do `hyprsome move monitor 2` or `hyprsome move monitor HDMI-A-2`

**Comment by sopa0** [comment]
so make it move from 1 to 2, from 2 to 3, from 3 to 1, yeah that's doable

**Comment by czettnersandor** [comment]
Perhaps this would work, no? https://github.com/sopa0/hyprsome/pull/3

I'm not sure if it's still applicable, but I'm using it like this without updates from this repo.
